packages/libraries/cli/src/base-command.ts

```diff
@@ -127,38 +139,34 @@ export default abstract class BaseCommand<T extends typeof Command> extends Comm
     }>;
 
     defaultValue?: TArgs[keyof TArgs] | null;
-    message?: string;
+    description: string;
     env?: string;
   }): NonNullable<GetConfigurationValueType<TKey>> | never {
-    if (args[key] != null) {
-      return args[key] as NonNullable<GetConfigurationValueType<TKey>>;
-    }
-
-    if (legacyFlagName && (args as any)[legacyFlagName] != null) {
-      return args[legacyFlagName] as any as NonNullable<GetConfigurationValueType<TKey>>;
-    }
-
-    // eslint-disable-next-line no-process-env
-    if (env && process.env[env]) {
-      // eslint-disable-next-line no-process-env
-      return process.env[env] as TArgs[keyof TArgs] as NonNullable<GetConfigurationValueType<TKey>>;
-    }
-
-    const userConfigValue = this._userConfig!.get(key);
-
-    if (userConfigValue != null) {
-      return userConfigValue;
-    }
+    let value: GetConfigurationValueType<TKey>;
 
-    if (defaultValue) {
-      return defaultValue;
+    if (args[key] != null) {
```

Comment:
These changes are so that empty strings dont fallback to the default.
I believe empty strings for a required argument indicates there is something wrong, like a variable wasnt set within a script correctly